src/layout/Titlebar/Titlebar.js

```diff
@@ -1,17 +1,10 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import './Titlebar.scss';
 
-export const Titlebar = ({ title }) => (
+export default ({ title }) => (
```

Comment:
I prefer previsous version. With Proptypes and named export.

Reply:
as we discussed, I'll use again Proptypes and named export

Reply:
done